### PR TITLE
Improve the accuracy of RETINA_PREFIX

### PR DESCRIPTION
**Pull request by finscn**
Sometimes ，this PR could fix https://github.com/pixijs/pixi.js/issues/3755 .
Maybe this PR is better than https://github.com/pixijs/pixi.js/pull/3756.

### CODE REVIEW

**Comment by bigtimebuddy** [comment]
I think this is a better approach.

**Comment by finscn** [comment]
@bigtimebuddy , I found the doc says : 
```
     * @type {RegExp|string}
```
But if  RETINA_PREFIX is string , the  `RETINA_PREFIX.exec` will take an error .

Is there  a fault in the doc ?

**Comment by bigtimebuddy** [comment]
Yes, seems like it should be a RegExp only.

File: src/core/settings.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ export default {
      * @example `@2x`
      * @default /@(.+)x/
```
Comment (staff0rd) [inline src/core/settings.js]: `@default` needs to be updated

Comment (finscn) [inline src/core/settings.js]: Done.

**Pull request merged**